### ISSUE TITLE
`cssText` does not updates

### ISSUE THREAD

**Issue by NV** [subtleGradient/Sheet.js#3]
```
var myStyleSheet = new Sheet("#selector { color:blue }")
myStyleSheet.cssRules[0].style.color = "green"
myStyleSheet.cssRules[0].style.cssText //'color:blue;'
```

I can fix it by calling `update()`

```
myStyleSheet.update()
myStyleSheet.cssRules[0].style.cssText //'color:green;'
```

but in the browser it works without update method (which actually doesn't exist for CSSStyleSheet).

I expected to see `CSSStyleRule` class to implement `cssText` getter, but I didn't found it. The code looks a bit tricky to extend at first glance. Some pointers would be helpful.

**Comment by subtleGradient** [comment]
Unfortunately getters aren't UniversalJS, so I don't use them. The only possible options are to have some sort of refresh method like update or to use a function for getting the cssText.

**Comment by NV** [comment]
`Object.prototype.__defineGetter__` is supported by latest version of V8 (and so Node and Chrome), JavaScriptCore (Safari), Firefox, Opera and IE9 dev preview. The only browser outside is IE < 9.

What about cssText as a function? It would be more efficient than `Sheet.prototype.update` which iterates all rules.

**Comment by subtleGradient** [comment]
Hmm… I guess I could make a getCssText method. Then I could also use a getter when it is supported by the environment. 
It might complicate the usage code a bit when used in browsers, but that migh not be too bad.

**Comment by NV** [comment]
I found another way. We could define [setProperty](http://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSSStyleDeclaration-setProperty) method and update cssText when setProperty called. It would work in IE and matches CSSOM spec.

**Comment by subtleGradient** [comment]
Good idea. There should still be a way to update it when any properties are modified directly. But I'd just put it into the docs and recommend people use the set method.
